azext_iot/operations/hub.py

```diff
@@ -146,6 +150,27 @@ def iot_device_create(
         login=login,
         auth_type=auth_type_dataplane,
     )
+    return _iot_device_create(
+        target, device_id, edge_enabled, auth_method, primary_key, secondary_key, primary_thumbprint,
```

Comment:
There are many attributes for the function, explicitly setting kwargs would be helpful